pontoon/api/urls.py

```diff
@@ -14,6 +14,61 @@
         r"^graphql/?$",
         csrf_exempt(GraphQLView.as_view(schema=schema, graphiql=True)),
     ),
+    path(
+        "api/new/",
+        include(
+            [
+                path(
+                    # Locales
+                    "locales/",
+                    views.LocaleListView.as_view(),
+                    name="locale-list",
+                ),
+                path(
+                    # Locale
+                    "locales/<str:code>",
+                    views.LocaleIndividualView.as_view(),
+                    name="locale-individual",
+                ),
+                path(
+                    # Projects
+                    "projects/",
+                    views.ProjectListView.as_view(),
+                    name="project-list",
+                ),
+                path(
+                    # Project
+                    "projects/<str:slug>",
+                    views.ProjectIndividualView.as_view(),
+                    name="project-individual",
+                ),
+                path(
+                    # Localizations
+                    "localizations/",
+                    views.ProjectLocaleListView.as_view(),
+                    name="localization-list",
+                ),
+                path(
+                    # Localization
+                    "localizations/<str:slug>/<str:code>",
```

Comment:
The API URLs ought to match the patterns used in Pontoon in general, which are primarily indexed on the locale. This should probably be something like:
```suggestion
                    # Localization
                    "locales/<str:code>/<str:slug>",
```

Reply:
Presuming that you meant `[code]/[slug]`, that could work, if we don't mind sharing the namespace between locales and all the other endpoints.